requirements.txt

```diff
@@ -12,7 +12,7 @@ flake8==2.6.2
 mccabe==0.5.0
 certifi==2016.2.28
 pyparsing==2.1.5
-backports.ssl-match-hostname==3.5.0.1
+backports.ssl_match_hostname==3.5.0.1
```

Comment:
I changed the dashes here to underscores, since that's what it's registered as in PyPI, but either way works.